src/simtools/applications/simulate_prod.py

```diff
@@ -122,15 +122,25 @@ def _parse(description=None):
         required=False,
         default=False,
     )
-    config.parser.add_argument(
-        "--sim_telarray_seeds",
+    sim_telarray_seed_group = config.parser.add_argument_group(
+        title="Random seeds for sim_telarray instrument setup",
+    )
+    sim_telarray_seed_group.add_argument(
+        "--sim_telarray_instrument_seeds",
         help=(
-            "Use the provided random seed for sim_telarray instrument setup."
-            "Used only for testing purposes for now."
+            "Random seed used for sim_telarray instrument setup. "
+            "If '--sim_telarray_random_instrument_instances is not set: use as sim_telarray seed. "
```

Comment:
Since the original sim_telarray seed option was changed, this description is not clear enough. I suggest to at least mention it is provided to as the `random_seed` sim_telarray parameter so one can search the sim_telarray documentation.  Maybe even copying a sentence or two from there would be helpful.

Reply:
Improved the help message (already very long...)